src/FieldEncryptProcessEntities.php

```diff
@@ -138,16 +120,14 @@ protected function process_field(\Drupal\Core\Field\FieldItemListInterface $fiel
 
     $field_type = $definition->get('field_type');
 
-    /**
-     * Filter out fields that do not have a defined map.
-     */
-    if (!in_array($field_type, array_keys($this->getFieldEncryptMap()))) {
+    // Filter out fields that do not have a defined encryption provider for
+    // their field type.
+    $supported_types = $this->providerPluginManager->getSupportedFieldTypes();
```

Comment:
I do think there may be value in "settings" but I want to discuss potential use cases.

I think it's much better to select the properties to encrypt, select an encryption profile, and drop the mic. ;)